src/Examples.Console/Examples.Console.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What changed here :eyes:

Reply:
Some weird char at the start. Prob VS auto format fixed it.
![image](https://user-images.githubusercontent.com/5938087/142241894-a0ebdc9f-fe26-4fd5-b844-3809eeca1a34.png)